[torch] Error generating `torch_hip_generated_int4mm.hip.obj` for `gfx90X` windows during torch builds

Currently getting the following error during `gfx90X` windows builds for torch:

```
_chunk_kernel<at::native::ALayout_RM<at::native::KReductionType::None>, at::native::BLayout_TC_int4<4, 128>, at::native::ALayout_RM<at::native::KReductionType::None>, 8, 8>'
2025-10-23T22:03:19.3242400Z   693 |     BLayout::template load<KTilesPerIteration>(
2025-10-23T22:03:19.3242719Z       |                       ^
2025-10-23T22:03:19.3243320Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:585:16: error: reference to __host__ function 'memcpy' in __device__ function
2025-10-23T22:03:19.3244000Z   585 |           std::memcpy(&out[i][j], &v, sizeof(bf16x2x4_u32));
2025-10-23T22:03:19.3244311Z       |                ^
2025-10-23T22:03:19.3245371Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:799:23: note: called by 'tinygemm_m16n8k16_chunk_kernel<at::native::ALayout_RM<at::native::KReductionType::None>, at::native::BLayout_TC_int4<4, 128>, at::native::ALayout_RM<at::native::KReductionType::None>, 8, 8>'
2025-10-23T22:03:19.3246628Z   799 |     BLayout::template load<kInnerKTiles>(
2025-10-23T22:03:19.3246994Z       |                       ^
2025-10-23T22:03:19.3247594Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:585:16: error: reference to __host__ function 'memcpy' in __device__ function
2025-10-23T22:03:19.3248382Z   585 |           std::memcpy(&out[i][j], &v, sizeof(bf16x2x4_u32));
2025-10-23T22:03:19.3248691Z       |                ^
2025-10-23T22:03:19.3249596Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:693:23: note: called by 'tinygemm_m16n8k16_chunk_kernel<at::native::ALayout_RM<at::native::KReductionType::None>, at::native::BLayout_TC_int4<8, 128>, at::native::ALayout_RM<at::native::KReductionType::None>, 8, 8>'
2025-10-23T22:03:19.3250609Z   693 |     BLayout::template load<KTilesPerIteration>(
2025-10-23T22:03:19.3250889Z       |                       ^
2025-10-23T22:03:19.3251396Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:585:16: error: reference to __host__ function 'memcpy' in __device__ function
2025-10-23T22:03:19.3252024Z   585 |           std::memcpy(&out[i][j], &v, sizeof(bf16x2x4_u32));
2025-10-23T22:03:19.3252377Z       |                ^
2025-10-23T22:03:19.3253573Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:799:23: note: called by 'tinygemm_m16n8k16_chunk_kernel<at::native::ALayout_RM<at::native::KReductionType::None>, at::native::BLayout_TC_int4<8, 128>, at::native::ALayout_RM<at::native::KReductionType::None>, 8, 8>'
2025-10-23T22:03:19.3254963Z   799 |     BLayout::template load<kInnerKTiles>(
2025-10-23T22:03:19.3255260Z       |                       ^
2025-10-23T22:03:19.3255981Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:585:16: error: reference to __host__ function 'memcpy' in __device__ function
2025-10-23T22:03:19.3256685Z   585 |           std::memcpy(&out[i][j], &v, sizeof(bf16x2x4_u32));
2025-10-23T22:03:19.3257019Z       |                ^
2025-10-23T22:03:19.3258128Z B:/src/torch/aten/src/ATen/native/hip/int4mm.hip:693:23: note: called by 'tinygemm_m16n8k16_chunk_kernel<at::native::ALayout_RM<at::native::KReductionType::None>, at::native::BLayout_TC_int4<2, 256>, at::native::ALayout_RM<at::native::KReductionType::None>, 8, 8>'
2025-10-23T22:03:19.3259534Z   693 |     BLayout::template load<KTilesPerIteration>(
2025-10-23T22:03:19.3259885Z       |                       ^
2025-10-23T22:03:19.3260367Z fatal error: too many errors emitted, stopping now [-ferror-limit=]
2025-10-23T22:03:19.3260909Z 2 warnings and 20 errors generated when compiling for gfx90a.
2025-10-23T22:03:19.3285663Z failed to execute:""C:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/lib/llvm/bin\clang.exe"   --offload-arch=gfx906 --offload-arch=gfx908 --offload-arch=gfx90a -O3  -c -x hip "B:/src/torch/aten/src/ATen/native/hip/int4mm.hip" -o "B:/src/torch/build/caffe2/CMakeFiles/torch_hip.dir/__/aten/src/ATen/native/hip/./torch_hip_generated_int4mm.hip.obj" --offload-compress -fclang-abi-compat=17 -DUSE_ROCM -D__HIP_PLATFORM_AMD__ -DUSE_FLASH_ATTENTION -DFLASHATTENTION_DISABLE_ALIBI -DFLASHATTENTION_DISABLE_SOFTCAP -DFLASH_NAMESPACE=pytorch_flash -DUNFUSE_FMA -DUSE_MEM_EFF_ATTENTION -DTORCH_HIP_BUILD_MAIN_LIB -DROCM_ON_WINDOWS -DROCM_VERSION=71000 -DTORCH_HIP_VERSION=701 -DUSE_LAYERNORM_FAST_RECIPROCAL -DONNX_ML=1 -DONNXIFI_ENABLE_EXT=1 -DONNX_NAMESPACE=onnx_torch -D_CRT_SECURE_NO_DEPRECATE=1 -DUSE_EXTERNAL_MZCRC -DMINIZ_DISABLE_ZIP_READER_CRC32_CHECKS -DEXPORT_AOTI_FUNCTIONS -DWIN32_LEAN_AND_MEAN -D_UCRT_LEGACY_INFINITY -DNOMINMAX -DUSE_MIMALLOC -DUSE_PROF_API=1 -DAT_PER_OPERATOR_HEADERS -D__HIP_PLATFORM_AMD__=1 -D__HIP_PLATFORM_AMD__ -DFMT_HEADER_ONLY=1 -fms-runtime-lib=dll -D__HIP_PLATFORM_AMD__=1 -DCUDA_HAS_FP16=1 -DUSE_ROCM -D__HIP_NO_HALF_OPERATORS__=1 -D__HIP_NO_HALF_CONVERSIONS__=1 -DTORCH_HIP_VERSION=701 -Wno-shift-count-negative -Wno-shift-count-overflow -DCAFFE2_USE_MIOPEN -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_HIP -std=c++17 -DHIPBLAS_V2 -DHIP_ENABLE_WARP_SYNC_BUILTINS -fms-extensions -Wno-ignored-attributes -fno-gpu-rdc -IB:/src/torch/build/aten/src -IB:/src/torch/aten/src -IB:/src/torch/build -IB:/src/torch -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IB:/src/torch/third_party/protobuf/src -IB:/src/torch/third_party/XNNPACK/include -IB:/src/torch/third_party/ittapi/include -IB:/src/torch/cmake/../third_party/eigen -IB:/src/torch/third_party/ideep/mkl-dnn/include/oneapi/dnnl -IB:/src/torch/third_party/ideep/include -IB:/src/torch/nlohmann -IB:/src/torch/INTERFACE -IB:/src/torch/third_party/nlohmann/include -IB:/src/torch/moodycamel -IB:/src/torch/INTERFACE -IB:/src/torch/third_party/concurrentqueue -IB:/src/torch/third_party/mimalloc/include -IB:/src/torch/aten/src/THH -IB:/src/torch/third_party/fmt/include -IB:/src/torch/aten/src -IB:/src/torch/build/caffe2/aten/src -IB:/src/torch/build/aten/src -IB:/src/torch/aten/src -IB:/src/torch/aten/src/ATen/.. -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IB:/src/torch/torch/include -IB:/src/torch/c10/hip/../.. -IB:/src/torch/build -IB:/src/torch/c10/../ -IB:/src/torch/build -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IB:/src/torch/torch/csrc/api -IB:/src/torch/torch/csrc/api/include -IB:/src/torch/third_party/protobuf/src -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include/hiprand -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include/rocrand -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IB:/src/torch/build/aten/src -IB:/src/torch/aten/src -IB:/src/torch/build -IB:/src/torch -IC:/home/runner/_work/_tool/Python/3.13.9/x64/Lib/site-packages/_rocm_sdk_devel/include -IB:/src/torch/third_party/protobuf/src -IB:/src/torch/third_party/XNNPACK/include -IB:/src/torch/third_party/ittapi/include -IB:/src/torch/cmake/../third_party/eigen -IB:/src/torch/third_party/ideep/mkl-dnn/include/oneapi/dnnl -IB:/src/torch/third_party/ideep/include -IB:/src/torch/nlohmann -IB:/src/torch/INTERFACE -IB:/src/torch/third_party/nlohmann/include -IB:/src/torch/moodycamel -IB:/src/torch/third_party/concurrentqueue -IB:/src/torch/third_party/mimalloc/include"
2025-10-23T22:03:19.3311759Z CMake Error at torch_hip_generated_int4mm.hip.obj.cmake:200 (message):
2025-10-23T22:03:19.3312253Z   Error generating file
2025-10-23T22:03:19.3312948Z   B:/src/torch/build/caffe2/CMakeFiles/torch_hip.dir/__/aten/src/ATen/native/hip/./torch_hip_generated_int4mm.hip.obj
2025-10-23T22:03:19.3313584Z 
2025-10-23T22:03:19.3313605Z 
```

--- a/build_tools/github_actions/amdgpu_family_matrix.py
+++ b/build_tools/github_actions/amdgpu_family_matrix.py
@@ -180,7 +180,6 @@ amdgpu_family_info_matrix_nightly = {
             "family": "gfx900",
             "fetch-gfx-targets": [],
             "build_variants": ["release"],
-            "expect_pytorch_failure": True,
         },
     },
     # gfx906/908/90a split into separate families - each has different instruction
@@ -200,7 +199,6 @@ amdgpu_family_info_matrix_nightly = {
             "family": "gfx906",
             "fetch-gfx-targets": [],
             "build_variants": ["release"],
-            "expect_pytorch_failure": True,
         },
     },
     "gfx908": {
@@ -217,7 +215,6 @@ amdgpu_family_info_matrix_nightly = {
             "family": "gfx908",
             "fetch-gfx-targets": [],
             "build_variants": ["release"],
-            "expect_pytorch_failure": True,
         },
     },
     "gfx90a": {
@@ -233,7 +230,6 @@ amdgpu_family_info_matrix_nightly = {
             "family": "gfx90a",
             "fetch-gfx-targets": [],
             "build_variants": ["release"],
-            "expect_pytorch_failure": True,
         },
     },
     "gfx101x": {

--- a/build_tools/github_actions/new_amdgpu_family_matrix.py
+++ b/build_tools/github_actions/new_amdgpu_family_matrix.py
@@ -412,8 +412,6 @@ amdgpu_family_info_matrix_all = {
                     "bypass_tests_for_releases": False,
                 },
             },
-            # TODO(#1927): Resolve error generating file `torch_hip_generated_int4mm.hip.obj`,
-            # to enable PyTorch builds
             "windows": {
                 "build": {
                     "build_variants": ["release"],
@@ -422,7 +420,6 @@ amdgpu_family_info_matrix_all = {
                     "run_tests": False,
                     "runs_on": {},
                     "fetch-gfx-targets": [],
-                    "expect_pytorch_failure": True,
                 },
                 "release": {
                     "push_on_success": False,
@@ -496,7 +493,6 @@ amdgpu_family_info_matrix_all = {
                     },
                     "fetch-gfx-targets": [],
                     "sanity_check_only_for_family": True,
-                    "expect_pytorch_failure": True,
                 },
                 "release": {
                     "push_on_success": False,

